chapter_6/ex6.rb

```diff
@@ -0,0 +1,22 @@
+#not sure what the commit here, all of it is self explanatory
```

Comment:
Please go through each line and explain it in English as if you were talking to me. For example, line 2 is assigning a number to the variable "types_of_people"